.github/workflows/generate_vectors.yml

```diff
@@ -0,0 +1,81 @@
+name: Run test vector generation
+
+defaults:
+  run:
+    shell: zsh {0}
+
+env:
+  TEST_PRESET_TYPE: "minimal"
```

Comment:
Does it matter?

Reply:
You're right, the variable isn't used so I can remove it 🗡️